EdFi.Tools.ApiPublisher.Connections.Api/Processing/Target/Blocks/PostResourceProcessingBlocksFactory.cs

```diff
@@ -264,8 +264,9 @@ public class PostResourceProcessingBlocksFactory : IProcessingBlocksFactory<Post
 
                                     // Clean up the JObject for POSTing against the target
                                     // Remove attributes not usable between API instances
-                                    missingItem.Remove("id");
-                                    missingItem.Remove("_etag");
+                                    //These lines have commented out because the same logic is in the HandlePostItemMessage method

```

Comment:
@CSR2017 
As I mentioned in a comment, this logic is included in the method HandlePostItemMessage, if I revert these lines it will cause an error on that method and it won't be able to send the request to the API

![image](https://user-images.githubusercontent.com/105452282/232861515-9fa195c1-9677-410b-82bb-440c77f31346.png)